Davidodari/POLITICO
# #163892093 Fix Static Site Name to Redirect to home page

### What Does this PR do?
- Fixes Static Page Name to a Url to the home page
- Adds gitignore for unneeded files

### Description of Task To Be Completed
Have The SIte Name redirect to a home page

### How Can This Be Manually Tested
1. Clone the repo and checkout to bg-fix-sitename-163892093
2. Open any page in a browser and click on site name on the top left corner, it should redirect to home page or if on the login page or sign up page should remain as it is
 
### Relevant PT Stories?
_#163892093_
